tools/extract-changelog.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# Written by MRDGH2821 <ask.mrdgh2821@outlook.com>
+# This code is licensed under the MIT license.
+
+
+import os
+import re
+import sys
+
+
+def extract_latest_changelog(folder_path):
+    changelog_file_path = os.path.join(folder_path, "CHANGELOG.md")
+    with open(changelog_file_path, "r") as f:
+        content = f.read()
+
+    # Find the latest version header
+    latest_version_header = re.search(r"^## v\d+\.\d+\.\d+$", content, re.MULTILINE)
+    if latest_version_header:
+        latest_version_header = latest_version_header.group()
+        # Find the section corresponding to the latest version
+        section = re.search(
+            rf"{latest_version_header}(.*?)(?=^##|\Z)",
+            content,
+            re.DOTALL | re.MULTILINE | re.S,
+        )
+        if section:
+            return section.group(0).strip()
+    return None
+
+
+if __name__ == "__main__":
+    if len(sys.argv) != 2:
```

Comment:
Take a second parameter that is the expected version number and only return if the latest changelog matches that version.  That way, if the file isn't updated, it won't erroneously include the changes for the previous version.

Or enhance it to return the changelog for _any_ specific version.

Reply:
Resolved in https://github.com/bcwhite-code/brians-blueprints/pull/27/commits/d372ceb88d47dc84c0a9a5d0a4be54295a47d8f3